docs/licenses.md

```diff
@@ -15190,7 +15190,7 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 ```
 
-### sass-embedded-darwin-arm64/dart-sass/src
+### sass-embedded-darwin-x64/dart-sass/src
```

Comment:
I guess the platform is flipping back and forth based on the developer's platform, who is running the license generation?

I'm guessing this shouldn't impact the release, but should this be looked at?

Reply:
yup, it seems so 🤷‍♂️